forgerock-core/src/main/java/org/forgerock/android/auth/biometric/BiometricAuth.kt

```diff
@@ -67,12 +70,8 @@ class BiometricAuth @JvmOverloads constructor(
      */
     private val description: String? = null,
 
-    /**
-     * The type of authenticator to be displayed.
-     * @return the authenticatorType as Int.
-     */
-    var authenticatorType: AuthenticatorType = AuthenticatorType.WEAK
```

Comment:
we need to test the authenticator app and make sure we didnt break the biometric after user receive the push notification . probably i will ask stoyan to do this test. OR i can do this as well

Reply:
I agree, this code is share with Authenticator App.